lib/Drush/CommandFiles/core/DrupliconCommands.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Drush\CommandFiles\core;
+
+class DrupliconCommands {
+
+  /**
+   * Print druplicon as post-command output.
+   *
+   * @hook post-command *
+   * @option $druplicon Shows the druplicon as glorious ASCII art.
+   */
+  public function druplicon($result, $argsAndOptions, $annotationData) {
+    if (isset($argsAndOptions['options']['druplicon'])) {
```

Comment:
@weitzman Do you hate `$argsAndOptions`?  I think I hate it.  I could split this into separate `$args` and `$options`.  It is tempting to try to make the new post-command hook look like the old post-command hook -- that is, have Drush call it with the same calling conventions as the primary command hook.  However, doing that is hard on hooks that want to attach to every command, as the parameter order can be different for different commands.  Additionally, we want to pass in options as a parameter rather than call a globally-scoped function, and we also have annotation data to provide.

Finally, note that the master branch does not use `isset` in its comparison; at the moment, 8.x is not providing default values for missing options.  I'll try to fix that next, so that we can have the same expectations in 8.x and master.